project/BuildDependencies/scripts/0_package.target-win10-x64.list

```diff
@@ -0,0 +1,42 @@
+; package.list should contain all file names of packages which are already in the "new" package format (the package contains the paths to the libraries
+; and headers relative to the XBMC root directory, stored in a zip container)
+; example sqlite-3.7.12.1-win32.7z:
+; -> (package name)\(relative path to file)
+; -> sqlite-3.7.12.1-win32\project\BuildDependencies\lib\sqlite3.lib
+; -> sqlite-3.7.12.1-win32\system\sqlite3.dll
+; -> ...
+;PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER!
+cpluff-0.1.3-win10-x64-v140.7z
+crossguid-fef89a4-win10-x64-v140.7z
+curl-7.52.1-win10-x64-v140.7z
+expat-2.2.0-win10-x64-v140.7z
+fmt-3.0.1-win10-x64-v140.7z
+freetype-2.8-win10-x64-v140.7z
+lcms2-2.8-win10-x64-v140.7z
+libaacs-0.9.0-win10-x64-v140.7z
+libass-0.13.6-win10-x64-v140.7z
+libbdplus-45c76dd-win10-x64-v140.7z
+libbluray-1.0.1-win10-x64-v140.7z
+libcdio-0.94-win10-x64-v140.7z
+libfribidi-0.19.7-win10-x64-v140.7z
+libiconv-1.15-win10-x64-v140.7z
+libmicrohttpd-0.9.55-win10-x64-v140.7z
+libnfs-1.11.0-win10-x64-v140.7z
+libplist-1.12-win10-x64-v140.7z
+libssh2-1.8.0-win10-x64-v140.7z
+libxml2-2.9.4-win10-x64-v140.7z
+libxslt-1.1.29-win10-x64-v140.7z
+lzo2-2.09-win10-x64-v140.7z
+mini_wdk-10.0.14393.0-win32.7z
+openssl-1.0.2k-win10-x64-v140.7z
+pcre-8.40-win10-x64-v140.7z
+pillow-4.2.1-win10-x64-v140.7z
+pycryptodome-3.4.7-win10-x64-v140.7z
+python-2.7.13-win10-x64-v140.7z
+rapidjson-1.1.0-win32.7z
+shairplay-ce80e00-win10-x64-v140.7z
+sqlite-3170000-win10-x64-v140.7z
+swig-3.0.10-win32.7z
```

Comment:
native package

Reply:
and? It's required only for generate Python bindings and should be executable on build host

Reply:
now I see why you have done it like that, it's the same for win32 and x64 (which is my fault)
https://github.com/xbmc/xbmc/blob/dd0dc92289281098eb28187eb134242ca1f3716f/project/BuildDependencies/scripts/0_package.target-x64.list#L42 https://github.com/xbmc/xbmc/blob/45874ebd3e57a897b1435d9bcfecbce258eb9926/project/BuildDependencies/scripts/0_package.target-win32.list#L45


correct place is https://github.com/xbmc/xbmc/blob/6e5b814/project/BuildDependencies/scripts/0_package.native-win32.list

Reply:
moved to native on all platforms